src/components/ContentSidebar/ActivityFeed/Avatar.js

```diff
@@ -39,6 +39,11 @@ class Avatar extends React.PureComponent<Props, State> {
      */
     getAvatarUrl() {
         const { user, getAvatarUrl }: Props = this.props;
+
+        if (!getAvatarUrl) {
+            return Promise.resolve(user.avatar_url).then(this.getAvatarUrlHandler);
```

Comment:
Also, according to our types avatar_url is an optional field, so we'll need to check that it exists as well.

Reply:
The promise is to match the promise return by the getAvatarUrl prop and to test the method easier. 

Since avatar_url is optional it actually matches how this.getAvatarUrl is structured since that already expects an optional parameter and updates the state accordingly